### PR TITLE
Add Reference to Background Materials in Deep Learning

### PR DESCRIPTION
**Pull request by astonzhang**
Dive into Deep Learning (D2L) encourages DL beginners to refer to TVM if they are interested in frontiers in DL such as DL compiler:

https://github.com/diveintodeeplearning/d2l-zh/blob/master/README.md
https://github.com/diveintodeeplearning/d2l-en/blob/master/README.md

Some contents in the TVM tutorials may implicitly assumes that readers know DL: in case that they don't, they may benefit from D2L.

### CODE REVIEW

**Comment by tqchen** [comment]
Thanks for the proposal,  it is indeed helpful to add reference materials for backgrounds. I had a list here: http://dlsys.cs.washington.edu/materials, can you update the link with the materials, as far as I understand, D2L is an upgraded version of "The Straight Dope"

**Comment by tqchen** [comment]
@yidawang we cannot because this is the restriction of the sphinx-gallery tool

**Comment by yidawang** [comment]
OK. Then we should not use the markdown format in the text. @astonzhang

**Comment by tqchen** [comment]
README.txt will be converted into ReStructuredText in sphinx-gallery, so it is fine to make use of the rst format. It is a fun requirement by sphinx-gallery.

**Comment by astonzhang** [comment]
To update: just now I was suggested to remove external links to our projects on D2L readme. The purpose is to be unbiased. We may think of a better (natural) way to raise awareness for our projects via D2L :)

**Comment by tqchen** [comment]
Can you also copy the two other links from http://dlsys.cs.washington.edu/materials to this PR as well, thanks

File: tutorials/README.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
 Tutorials
 =========
-This page contains the tutorials about TVM.
+This page contains the tutorials about TVM. For background knowledge of deep learning, please refer to the interactive book `Dive into Deep Learning <https://d2i.ai>`__, the `Deep Learning book <http://www.deeplearningbook.org/>`__, or `Stanford CS231n course <http://cs231n.stanford.edu/>`__.
```
Comment (vinx13) [inline tutorials/README.txt]: typo: https://d2l.ai

Comment (astonzhang) [reply tutorials/README.txt]: We just updated the url to this one. It may take a while to take effect

**Comment by tqchen** [comment]
I think a bit more about this. Given that deep learning is a well-received topic and most likely readers can already find materials by searching online, we may not have to include links to materials here. As the project's goal is not to curate a comprehensive list, the suggested list could never-the-less limited.

**Comment by astonzhang** [comment]
That makes sense. Closing this pr.

**Pull request closed without merging**